recipe/build.sh

```diff
@@ -37,6 +37,9 @@ export OMPI_MCA_plm_rsh_agent=sh
 # Only pw, cp, and unit tests are safe to run when using cmake curently (to fix in later releases)
 #make test
 # there are known test failures that will be addressed later
+# disable test for aarch64 for now it takes too long and timeout of build
+if [[ $(lscpu | awk '/Architecture:/{print $2}') != "aarch64" ]]; then 
 ctest -L "pw|cp|unit" -LE epw --output-on-failure  || true
```

Comment:
Can you run at least _some_ of the tests?

I don't know which ones make most sense, but "unit" sounds like they are fast

Reply:
<img width="606" alt="image" src="https://user-images.githubusercontent.com/8831685/234955552-9b2e0149-a4a8-4dc2-acbd-7dff49779b48.png">

I think the 60s more should be enough. I'll try to run it once more, if it still not able to finish all, let's just keep it without tests?

Reply:
Good, all tests passed this time. Whereas it is a problem that the next time the test may fail again.